plugin/othttp/handler_test.go

```diff
@@ -100,3 +100,40 @@ func TestBasicFilter(t *testing.T) {
 		t.Fatalf("got %q, expected %q", got, expected)
 	}
 }
+
+func TestWithOperationNamer(t *testing.T) {
+	rr := httptest.NewRecorder()
+
+	var id uint64
+	tracer := mocktrace.MockTracer{StartSpanID: &id}
+
+	h := NewHandler(
+		http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if _, err := io.WriteString(w, "hello world"); err != nil {
+				t.Fatal(err)
+			}
+		}), "",
+		WithTracer(&tracer), WithOperationNamer(func(_ *http.Request) string { return "test_handler" }))
```

Comment:
I don't see anything testing whether the operation name specified here is actually applied.